docs/meshstack.kubernetes.landing-zones.md

```diff
@@ -0,0 +1,17 @@
+---
+id: meshstack.kubernetes.landing-zones
+title: Landing Zones
+---
+
+The Kubernetes Landingzone can be used to setup shared Namespaces which are created during replication for each meshProject.
```

Comment:
```suggestion
The Kubernetes Landing Zone can be used to setup shared namespaces which are created during replication for each meshProject.
```

i think we never write landingzone in one word

Reply:
but in general, what does that sentence mean? Shared namespaces sounds strange to me. Isn't it more like we create dedicated namespaces for meshProjects in a shared AKS cluster, but isolation between namespaces is still given?

Reply:
Yeah true. I just needed an introduction sentence. It was rather bad. Changed it too: `
The Kubernetes Landing Zone can be used to configure how Namspaces are created inside a shared Kubernetes Cluster. `